apps/src/StudioApp.js

```diff
@@ -1616,8 +1612,9 @@ StudioApp.prototype.handleEditCode_ = function (options) {
   }
 
   // droplet may now be in code mode if it couldn't parse the code into
-  // blocks, so update the UI based on the current state:
-  this.onDropletToggle_();
+  // blocks, so update the UI based on the current state (don't autofocus
+  // if we have already created an instructionsDialog at this stage of init)
+  this.onDropletToggle_(!!this.instructionsDialog);
```

Comment:
Just curious, what's your preference between `Boolean(x)` and `!!x` for boolean coercion?